tests/mol-tts_multi/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title>Multiple text-to-speech fallback for MO</title>
+</head>
+<body>
+    <p>Test passes (i.e., the Reading System correctly falls back on Text-to-Speech when no audio file is present) if</p>
+    <ol>
+        <li>The Reading System provides the user with a way to turn text reading on</li>
+        <li>If turned on, the extract on the next page is read aloud by the Reading System using Text-to-Speech</li>
+        <li>If the Reading System implements the association with style information, the text being read is highlighted with a green background and the rest of the text is grayed out</li>
```

Comment:
Does the "if" here imply that the test can still pass even if this last case fails, because the RS doesn't support CSS? If so I think this should be a note or something, not part of the list, and should be clearly marked as not required to pass.

Reply:
Also, the language choice here is a bit weird (stilted?). Can you say something like "If the Reading System implements styling for media overlays, ..."

Reply:
> Does the "if" here imply that the test can still pass even if this last case fails, because the RS doesn't support CSS?

Well... it is a gray area, in fact (the same terminology is there for a number of other tests). Yes, support for CSS is optional; however *if* the RS implements it, then the described behaviour is the required one. Ie, the test fails if it does not really do it the way it described.

For the sake of consistency with around 4-5 other tests, let us leave it as is.